src/Screens/OffsiteRec/OffsiteRec.jsx

```diff
@@ -0,0 +1,142 @@
+        </div>
+        <div className="test">
+          <p className="website-title">City of Snohomish</p>
+          <p className="">Antique Capital of the Northwest Over 100 Antique Stores</p>
+        </div>
+        <div className="test">
+          <p className="website-title">Evergreen State Fairgrounds</p>
+          <a href="http://www.evergreenfair.org/"target="_blank" rel="noopener noreferrer" className="website-link">www.evergreenfair.org</a>
+        </div>
+        <div className="test">
+          <p className="website-title">Hot Air Ballon Rides</p>
+          <a href="http://www.airialballoon.com/"target="_blank" rel="noopener noreferrer" className="website-link">www.airialballoon.com</a>
+        </div>
+        <div className="test">
+          <p className="website-title">Paved Walking and Biking Trail</p>
+          <p className="">Centennial Trail in Snohomish</p>
+        </div>
+        <div className="test">
+          <p className="website-title">Craft Supplies and Classes</p>
+          <a href="http://www.bfranklincrafts.com/"target="_blank" rel="noopener noreferrer" className="website-link">www.bfranklincrafts.com</a>
+        </div>
+        <div className="test">
+          <p className="website-title">Golf Course</p>
+          <a href="http://www.snohomishgolfcourse.com/" target="_blank" rel="noopener noreferrer" className="website-link">www.snohomishgolfcourse.com</a>
+        </div>
+        <div className="test">
+          <p className="website-title">Fish Hatchery</p>
+          <p className="">Wallace River Hatchery, Sultan, WA</p>
+        </div>
+        <div className="test">
+          <p className="website-title">2 Skateboard Parks</p>
+          <p className="">Monroe Board and Blade City Park</p>
+          <p className="">Averill Youth Complex - Snohomish (Tillicum Kiwanis Playground)</p>
+        </div>
+        <div className="test">
+          <p className="website-title">Indoor Swimming Pools</p>
+          <a href="https://ymca-snoco.org/our-locations/monroesky-valley-ymca/" target="_blank" rel="noopener noreferrer" className="website-link">ymca-snoco.org</a>
+          <a href="https://www.sno.wednet.edu/Domain/24"target="_blank" rel="noopener noreferrer" className="website-link">Snohomish Aquatic Center</a>
+        </div>
+        <div className="test">
+          <p className="website-title">Outdoor Swimming Areas</p>
+          <p className="">Flowing Lake Park ($5.00 per car)</p>
+          <p className="">Lake Tye Park (Monroe)</p>
+        </div>
+        </div>
+        <div className="right">
+        <div className="test">
+          <p className="right-time">Within 30 Minutes of The Homestead</p>
+        </div>
+        <div className="test">
+          <p className="website-title">Nurseries Open to the Public</p>
+          <a href="http://www.molbaks.com/" target="_blank" rel="noopener noreferrer" className="website-link">www.molbaks.com</a>
```

Comment:
These links (and other groups in this page as well) appear in UI as a single line, just FYI.

Reply:
Ok thanks for catching that. I will make sure that gets fixed when I do my next pull request.